### ISSUE TITLE
Can not compile with --advance 5: __future__ error

### ISSUE THREAD

**Issue by usmanov-danil** [dashingsoft/pyarmor#758]
1. On MacOS 12.3.1 run pyarmor to obfuscate the script inside the docker container
```
$ PYTHONOPTIMIZE=2 pyarmor obfuscate --advanced 5 --exclude core/internal/code/api/streamer.py --src="." -r --output=/new_app entry.py 
...
#22 21.24 INFO     Generating spp code for <frozen core.internal.code.models.file_model> ...
#22 21.25 INFO     Generate module spp code successfully
#22 21.25 INFO     Compiling spp code ...
#22 21.30 INFO     Compile spp code successfully
#22 21.34 INFO          core/internal/code/models/pipeline.py -> ../new_app/core/internal/code/models/pipeline.py
#22 21.34 INFO     Generating spp code for <frozen core.internal.code.models.pipeline> ...
#22 21.34 ERROR    from __future__ imports must occur at the beginning of the file (<frozen core.internal.code.models.pipeline>, line 1)
------
```

**Comment by jondy** [comment]
I'll check this.

One solution is let spp mode ignore this file by adding the following comment in the header
```python
# pyarmor options: no-spp-mode
```

**Comment by usmanov-danil** [comment]
Thank you for the reply. For now I use this comment yet.

Also, I forgot to add, that Python 3.7 was used.

**Comment by jondy** [comment]
Fixed in v7.5.1
